tasks/src/image-to-text/data.ts

```diff
@@ -0,0 +1,58 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from 12M image-text of Reddit",
+			id:          "red_caps",
+		},
+        {
+			// TODO write proper description
+			description: "Dataset from images generated by Stable diffusion on Discord",
+			id:          "poloclub/diffusiondb",
```

Comment:
Can we have this one instead? https://huggingface.co/datasets/conceptual_captions
This is an official benchmark by Google AI for the task that has 3.3M images with a paper and a leaderboard.